docs/services/gcp-services-guide.md

```diff
@@ -153,19 +130,20 @@ acquired from GCP:
     spec:
       credentialsSecretRef:
         name: gcp-provider-creds
+        namespace: crossplane-system
         key: credentials.json
       projectID: $PROJECT_ID
     EOF
 
-    kubectl -n $INFRA_NAMESPACE apply -f gcp-provider.yaml
+    kubectlapply -f gcp-provider.yaml
     unset BASE64ENCODED_GCP_PROVIDER_CREDS # we don't need this anymore.
     ```
 
 * Verify GCP provider was successfully registered by the crossplane
 
   ```bash
-  kubectl -n $INFRA_NAMESPACE get providers.gcp.crossplane.io
-  kubectl -n $INFRA_NAMESPACE get secrets
+  kubectl get providers.gcp.crossplane.io
+  kubectl -n crossplane-system get secrets
   ```
 
 #### Cloud-Specific Resource Classes
```

Comment:
We can probably drop the "cloud-specific" terminology now that there's only one form of resource class.